thumbor/requirements.txt

```diff
@@ -1,26 +1,11 @@
 envtpl==0.6.0
-circus==0.15.0
-circus-web==1.0.0
-boto==2.48.0
-botocore==1.12.253
-dateutils==0.6.6
-numpy==1.14.3
-remotecv==2.2.2
-opencv-engine==1.0.1
-thumbor==6.7.5
+remotecv==3.0.0
+boto==2.49.0
 tornado-botocore==1.5.0
-tc-aws==6.2.15
-tc-core==0.4.1
-tc-shortener==0.2.2
-raven==6.7.0
-thumbor-memcached==5.1.0
-tc-mongodb==5.1.0
-tc-redis==1.0.1
-thumbor-cloud-storage==3.0.0
-pgmagick==0.7.4
-graphicsmagick-engine==0.1.1
+python-dateutil
+dateutils==0.6.12
+shortuuid==1.0.8
+redis==3.5.3
+raven==6.10.0
 cairosvg==1.0.22
```

Comment:
can we update cairosvg to `CairoSVG>=2.5.1` ? it seems like this version has a security vulnerability?

Reply:
I guess... We can definitely try it.